.eslintrc.cjs

```diff
@@ -365,7 +365,7 @@ module.exports = {
         'website/**',
         '**/__benchmarks__/**',
         '**/__tests__/**',
-        'packages/jest-types/**/*',
+        '**/__typetests__/**',
```

Comment:
Strange to see `jest-types` here. Might be there was some limitation, but now lint is passing (at least locally).

It feels right adding `**/__typetests__/**` to the list, or?

Reply:
I don't remember the context here, but as long as CI is happy, I'm happy 😃